RecoPixelVertexing/Configuration/python/customizePixelTracksSoAonCPU.py

```diff
@@ -24,6 +24,46 @@ def customizePixelTracksSoAonCPU(process) :
 
   return process
 
+def customizePixelTracksSoAonCPUForWF(process) :
+
+  process.load('RecoLocalTracker/SiPixelRecHits/siPixelRecHitHostSoA_cfi')
+  process.load('RecoPixelVertexing.PixelTriplets.caHitNtupletCUDA_cfi')
+  process.load('RecoPixelVertexing.PixelVertexFinding.pixelVertexCUDA_cfi')
+  
+  process.CUDAService = cms.Service("CUDAService")
+  process.CUDAService. enabled = cms.untracked.bool(False)
+  
+  process.pixelTrackSoA = process.caHitNtupletCUDA.clone()
+  process.pixelTrackSoA.onGPU = False
+  process.pixelTrackSoA.pixelRecHitSrc = 'siPixelRecHitsPreSplitting'
+  process.pixelVertexSoA = process.pixelVertexCUDA.clone()
+  process.pixelVertexSoA.onGPU = False
+  process.pixelVertexSoA.pixelTrackSrc = 'pixelTrackSoA'
+
+  process.load('RecoPixelVertexing.PixelTrackFitting.pixelTrackProducerFromSoA_cfi')
+  process.pixelTracks = process.pixelTrackProducerFromSoA.clone()
+  process.load('RecoPixelVertexing.PixelVertexFinding.pixelVertexFromSoA_cfi')
+  process.pixelVertices = process.pixelVertexFromSoA.clone()
+  process.pixelTracks.pixelRecHitLegacySrc = 'siPixelRecHitsPreSplitting'
+
+  process.siPixelRecHitsPreSplitting = process.siPixelRecHitHostSoA.clone()
```

Comment:
Is there any concern that both module instances (`siPixelRecHitsPreSplitting` and `siPixelRecHitHostSoA`) would be run?

Reply:
I dumped the proccess in a single python file ([here](https://raw.githubusercontent.com/AdrianoDee/warehouse/master/configs/dumped_pixel_cpu.py)) and while it is added to the proccessit is not further inserted in any Task (or Sequence, Path ...) and ultimately in the schedule. Am I missing something? 

![dep](https://user-images.githubusercontent.com/16901146/94345928-8df16a00-0029-11eb-9603-7c4991f4527e.png)

In any case,  would it make more sense to have something like

``` 
from RecoLocalTracker.SiPixelRecHits.siPixelRecHitHostSoA_cfi import * 
process.siPixelRecHitsPreSplitting = siPixelRecHitHostSoA.clone()
```


which, dumped, would be like [this](https://raw.githubusercontent.com/AdrianoDee/warehouse/master/configs/dumped_pixel_cpu_2.py). This apply also to all the other loaded and then cloned modules.

Reply:
> I dumped the proccess in a single python file ([here](https://raw.githubusercontent.com/AdrianoDee/warehouse/master/configs/dumped_pixel_cpu.py)) and while `siPixelRecHitHostSoA` is defined in the proccess it is not further inserted in any Task (or Sequence, Path ...) and ultimately in the schedule

Ah, right, I missed that e.g. in `customizePixelTracksSoAonCPU()` the `siPixelRecHitHostSoA` is explicitly added to the `process.reconstruction_step`. Thanks.

> In any case, would it make more sense to have something like
> 
> ```
> from RecoLocalTracker.SiPixelRecHits.siPixelRecHitHostSoA_cfi import * 
> process.siPixelRecHitsPreSplitting = siPixelRecHitHostSoA.clone()
> ```

If the intention is never to use the module loaded into the `process`, it would indeed be more clear to never `load()` it, but `import`+`clone()`+assign directly (even if technically lone module declarations in the `process` are not harmful).

Reply:
I moved everything to `import` + `clone()` for clearness.